npm/generate.js

```diff
@@ -39,10 +39,15 @@ async function genPlatformPackage() {
     } else {
         name = `${cpu}-${os}`;
     }
+
+    const packageJson = await fs.readFile(resolve(__dirname, "./package.json"), "utf8");
+    const basePackage = JSON.parse(packageJson);
+    delete basePackage['dependencies']
+
     const platformPackage = {
+        ...basePackage,
```

Comment:
This could have unintended behaviours. For eg: `private: true`. We should pick the fields we care about and only add those.

Reply:
@tusharmath Fixed